test/unit/base.test.js

```diff
@@ -40,6 +41,21 @@ describe('CONVERT FUNCTION TESTS ', function() {
         done();
       });
     });
+    it('should generate collection for a custom content type headers requests.' +
+      specPath4, function(done) {
+      var openapi = fs.readFileSync(specPath4, 'utf8');
+      Converter.convert({ type: 'string', data: openapi }, { schemaFaker: true }, (err, conversionResult) => {
+        expect(err).to.be.null;
+        expect(conversionResult.result).to.equal(true);
+        expect(conversionResult.output.length).to.equal(1);
+        expect(conversionResult.output[0].type).to.equal('collection');
+        expect(conversionResult.output[0].data).to.have.property('info');
+        expect(conversionResult.output[0].data).to.have.property('item');
+        expect(conversionResult.output[0].data.item[0]).to.have.property('response');
+        expect(conversionResult.output[0].data.item[0].response[0]).to.have.property('body');
```

Comment:
need to do more than check for presence of properties. are these tests failing without the fix?